integration/simulation/output_stats.go

```diff
@@ -90,16 +90,16 @@ func (o *OutputStats) incrementStats(block *types.Block, l1Node ethadapter.EthCl
 
 		switch l1Tx := t.(type) {
 		case *ethadapter.L1RollupTx:
-			r, err := common.DecodeRollup(l1Tx.Rollup)
+			_, err := common.DecodeRollup(l1Tx.Rollup)
 			if err != nil {
 				testlog.Logger().Crit("could not decode rollup.", log.ErrKey, err)
 			}
-			if l1Node.IsBlockAncestor(block, r.Header.L1Proof) {
-				o.l2RollupCountInL1Blocks++
-				for _, batch := range r.Batches {
-					o.l2RollupTxCountInL1Blocks += len(batch.TxHashes)
-				}
-			}
+			//if l1Node.IsBlockAncestor(block, r.Header.L1Proof) {
```

Comment:
Do we permanently retire this logic? If not we need a todo here (or ticket)

Reply:
there is a ticket. 
The one we raised after the retro to bring this chain validation logic up to date with the new protocol, and to make it runnable against any chain. Not just the sims